packages/firebase_crashlytics/example/lib/main.dart

```diff
@@ -11,9 +13,14 @@ void main() {
 
   // Pass all uncaught errors to Crashlytics.
   FlutterError.onError = (FlutterErrorDetails details) {
```

Comment:
Might be simpler here to just say `FlutterError.onError = Crashlytics.instance.recordFlutterError;`